tidepods/sentinel2.py

```diff
@@ -444,11 +446,21 @@ def write_tide_values(tide_values, plist, level):
 
     mem_file = fiona.MemoryFile()
     ms = mem_file.open(crs=from_epsg(4326), driver="ESRI Shapefile", schema=pts_schema,)
+   
+    out_name_points = outfile.split('\\')[-1][:-4]+'.shp'
```

Comment:
Try using pathlib.Path to handle paths. Make sure that Linux/Windows specific paths can be parsed and also that your path handling is not specific to the ones you gave as example. Check this out https://github.com/chris1610/pbpython/blob/master/extras/Pathlib-Cheatsheet.pdf

Reply:
Done